functions.php

```diff
@@ -17,6 +17,39 @@
 
 // @todo move this somewhere else
 
+add_action('all_admin_notices', 'check_required_plugins');
+
+function check_required_plugins()
+{
+    $required_plugins_slugs = [
+        'advanced-custom-fields/acf.php' => [
```

Comment:
Instead `key/value` array create array of objects:

```php
$required_plugins = [
        [
          [
 'name' => "ACF",
 'url' => 'https://www.advancedcustomfields.com/',
 'slugs' => ['advanced-custom-fields/acf.php', 'advanced-custom-fields-pro/acf.php']
          ]
          ], // ...
```

Reply:
Can we assume the slugs will always be either one or two (normal or pro) for any plugin possible or should we write it as it's possible to be more than two slug per plugin? because if we can limit it to two, we can avoid two nested loop in here and the code can be cleaner.